packages/widgetbook_core/lib/src/navigation_tree/models/navigation_node.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:flutter/material.dart';
+import 'package:freezed_annotation/freezed_annotation.dart';
+import 'package:widgetbook_core/src/navigation_tree/enums/navigation_node_type.dart';
+
+part 'navigation_node.freezed.dart';
+
+@freezed
+class NavigationNode with _$NavigationNode {
```

Comment:
I think the class names in general in this PR might be improved. What do you think? @jenshor

Reply:
That's a great idea. I usually do the `data` suffix in these cases but I hesitated here because of the length, but I agree that consistency is more important. Will make the change